Add Account - Reconnect explanation (#435)
owncloud/docs-client-desktop: modules/ROOT/pages/using.adoc

image::using/client-7.png[The Overflow Menu, width=250,pdfwidth=60%]

=== Account Reconnecting

> Synchronisation happens on a regular basis and connects to the server configured every view minutes. A connection loss can occur because of unplugging the network cable or switching to a new wirelss network. In such a case and if one wants to immediatley restart syncing for an account and not wait for the automatical reconnect, a manual force reconnect button becomes available. This button becomes only selectable if the Desktop app is in reconnection state.

Reviewer: ```suggestion
Synchronization happens on a regular basis and connects to the server configured every view minutes. A connection loss can occur when the network cable is unplugged or when switching to a different wireless network. In such a case a manual force reconnect button becomes available in case you want to immediately restart syncing for an account and not wait for an automatic reconnect. This button only becomes available if the Desktop app is in reconnection state.
```

> Synchronization happens on a regular basis and connects to the server configured every view minutes. A connection loss can occur when the network cable is unplugged or when switching to a different wireless network. In such a case a manual force reconnect button becomes available in case you want to immediately restart syncing for an account and not wait for an automatic reconnect. This button only becomes available if the Desktop app is in reconnection state.

Reviewer 2: ```suggestion
Synchronization happens on a regular basis and connects to the server every few minutes. A connection loss can occur when the network cable is unplugged or when switching to a different wireless network. In such a case a manual force reconnect button becomes available in case you want to immediately restart syncing for an account and not wait for an automatic reconnect. This button only becomes available if the Desktop app is in reconnection state.
```

Author: Thanks for catching 😅
